css/styles.css

```diff
@@ -2,678 +2,526 @@
 /* stylelint-disable no-missing-end-of-source-newline */
```

Comment:
Please don't disable any linters. They are important for an error-free standardized codebase. These linter can be fixed automatically if you set up linter on your local and `npx stylelint --fix` command on your terminal. Check the setup instructions from this [link](https://github.com/microverseinc/linters-config/tree/master/html-css).

Reply:
Done